draft-ietf-mls-architecture.md

```diff
@@ -431,46 +445,66 @@ the group.
 
 ### Delivery of messages and attachments {#delivery-guarantees}
 
-The main responsibility of the Delivery Service is to ensure delivery
-of messages. Specifically, we assume that Delivery Services provide:
-
-* Reliable delivery: when a message is provided to the Delivery Service,
-  it is eventually delivered to all clients.
-
-* In-order delivery: messages are delivered to the group
-  in the order they are received by the Delivery Service
-  and in approximately the order in which they are sent
-  by clients. The latter is an approximate guarantee because
-  multiple clients may send messages at the same time
-  and so the Delivery Service needs some latitude in enforcing
-  ordering across clients.
-
-* Consistent ordering: the Delivery Service must ensure that all clients
-  have the same view of message ordering for cryptographically
-  relevant operations. This means that the Delivery Service MUST enforce
-  global consistency of the ordering of group operation messages.
-
-Note that the protocol provides three important pieces of information
+The main responsibility of the Delivery Service is to ensure delivery of
+messages. Some MLS messages need only be delivered to some members of group
+(e.g., the message initializing a new member's state), while others need to be
+delivered to all members.  The Delivery Service may enable these delivery
+patterns via unicast channels (sometimes known as "client fanout"), broadcast
+channels ("server fanout"), or a mix of both.
+
+For the most part, MLS does not require the Delivery Service to deliver messages
+in any particular order.  The one requirement is that because an MLS group has a
+linear history, the members of the group must agree on the order in which
+changes are applied.  Concretely, the group must agree on which MLS Commit
+messages to apply.  There are a variety of ways to achieve this agreement, but
+most of them rely on some help from the Delivery Service.  For example, if a
```

Comment:
If we decide to cast the DS as an abstract layer rather than a service (as done below), we could mandate that the DS does actually have to deal with the problem of deciding which commit to merge.